gcc/rust/checks/errors/borrowck/rust-bir-builder-expr-stmt.cc

```diff
@@ -281,7 +281,46 @@ ExprStmtBuilder::visit (HIR::CallExpr &expr)
 
 void
 ExprStmtBuilder::visit (HIR::MethodCallExpr &expr)
-{}
+{
+  // TODO : need ExprStmtBuilder::visit_expr (HIR::PathExprSegment)
+  PlaceId fn = visit_expr (expr.get_method_name ());
```

Comment:
The trick for this is to look into the gccrs backend and check how Philip is already doing it. That you need here is at: gcc\rust\backend\rust-compile-expr.cc:1281

Note that the back-end needs to handle way more then we need to do here. He only care about the flow of data and its types.